vital/bindings/c/types/bounding_box.cxx

```diff
@@ -102,7 +120,7 @@ void vital_bounding_box_destroy( vital_bounding_box_t* bbox)
 TYPE vital_bounding_box_ ## NAME( vital_bounding_box_t *bbox )          \
 {                                                                       \
   STANDARD_CATCH(                                                       \
-    "C::bounding_box::" # NAME, 0,                                      \
```

Comment:
This name change is inconsistent with the rest of Vital.  I'm not arguing that it makes total sense, but other classes use `C::class_name::` to make it clear that the exception is coming out of the C bindings layer.  If we are changing that convention I think it needs to be done globally, and I would not do that on this PR.